types/node/process.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module "process" {
+    export = process;
```

Comment:
Nit: Shouldn't this file contain the actual process definitions and the global should reference this?